app/views/dishes/_form.html.haml

```diff
@@ -1,11 +1,25 @@
--# frozen_string_literal: true
-= simple_form_for(@dish) do |f|
-  = f.error_notification
-  = f.error_notification message: f.object.errors[:base].to_sentence if f.object.errors[:base].present?
-
-  .form-inputs
-    = f.input :name
-    = f.input :price
-
-  .form-actions
-    = f.button :submit
+%form
+  .row.mb-3
+    %label.col-sm-2.col-form-label{:for => "@Name"} Name
+    .col-sm-10
+      %input#inputEmail3.form-control{:type => "name"}/
```

Comment:
Don't use inputEmail3 or something. Use good names which describe you point.
The same with hash syntax